src/TTGamesExplorerRebirthUI/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
You can remove this file since it's just a leftover from your personnal settings